structures/README.adoc

```diff
@@ -10,10 +10,10 @@ Explanations::
 define for which use case to use roles, playbooks, potentially workflows (in Ansible Controller/Tower/AWX), and how to split the code you write.
 
 Rationale::
-especially when writing automation in a team, it is important to have a certain level of consistence and make sure everybody has the same understanding.
+especially when writing automation in a team, it is important to have a certain level of consistency and make sure everybody has the same understanding.
 By lack of doing so, your automation becomes unreadable and difficult to grasp for new members or even for existing members.
 +
-This structure will also help you to have a consistent level of modelization so that re-usability becomes easier.
+This structure will also help you to have a consistent level of modularization so that re-usability becomes easier.
```

Comment:
i think _modelization_ is actually correct here but I agree that it is an uncommon way of putting it. How about changing it to

```suggestion
Following a consitent structure will increse re-usability.
```

Reply:
@joeashcraft could you please have another look at this so we can approve it next week.

Reply:
@mophahr I like yours better! It much easier to read. I'll copy your wording and fix a small typo in an update to this PR.